From dd267942253088b7d21402057e3b151a860f861b Mon Sep 17 00:00:00 2001
From: Parya Jafari <parya@interaptix.com>
Date: Wed, 11 Jan 2023 15:03:57 -0500
Subject: [PATCH] Patch embree support ARM

---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 52363bb..1b795b2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -242,6 +242,7 @@ IF(DEFINED EMBREE_ISA_AVX512SKX)
 ENDIF()
 
 TRY_COMPILE(COMPILER_SUPPORTS_ARM_NEON "${CMAKE_BINARY_DIR}" "${PROJECT_SOURCE_DIR}/common/cmake/check_arm_neon.cpp")
+set(COMPILER_SUPPORTS_ARM_NEON TRUE)
 
 IF (CMAKE_CXX_COMPILER_WRAPPER STREQUAL "CrayPrgEnv")
   SET(EMBREE_MAX_ISA "DEFAULT" CACHE STRING "Selects highest ISA to support.")
-- 
2.39.0

